style.css

```diff
@@ -0,0 +1,155 @@
+body {
+  margin: 0;
+  padding: 0;
+  min-width: 375px;
+  max-width: 768px;
+  background: #1c1a19;
+}
+
+/* nav container */
+
+.navContainer {
+  background: #3e3c3c;
+  height: 48px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 48px;
+}
+
+nav ul {
+  display: flex;
+}
+
+nav .logo {
+  font-family: $'Roboto';
```

Comment:
- [ ] The usage of fonts are not correct in your project. Kindly fix every font. Here is a demo [Link](https://www.loom.com/share/9912309eebdb46b698a3243a1c92e2c3) how to correctly use a font.